### PR TITLE
H-12 Mars viewer MobX homework

### PR DESCRIPTION
**Pull request by elegorod**

### CODE REVIEW

File: homeworks/12-mars-viewer-mobx/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_NASA_API_KEY=EJxNr6OrEqS7HfnoIlhnJRZS4DtHFQoLDyLafH62
```
Comment (dex157) [inline homeworks/12-mars-viewer-mobx/.env:1]: Good idea!

File: homeworks/12-mars-viewer-mobx/src/mars/heart-selected.svg [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 66.911 66.911" style="enable-background:new 0 0 66.911 66.911;" xml:space="preserve">
+<path style="fill:#E34326;" d="M66.911,22.831c0-10.563-8.558-19.122-19.118-19.122c-5.658,0-10.721,2.473-14.223,6.377  c-0.037,0.043-0.076,0.085-0.113,0.128c-3.5-3.98-8.618-6.505-14.334-6.505C8.561,3.709,0.005,12.268,0,22.831  c0,5.834,2.629,11.059,6.758,14.565H6.751l27.104,25.806l26.308-25.806h-0.012C64.279,33.89,66.911,28.669,66.911,22.831z"/>
+<g>
```
Comment (dex157) [inline homeworks/12-mars-viewer-mobx/src/mars/heart-selected.svg]: Is it a bag? )

**Review by dex157 — COMMENTED** [review]
Good work! The clean solution, clean code, glad to see & check )

File: homeworks/12-mars-viewer-mobx/src/mars/heart-selected.svg (hunk 2 above)
Comment (elegorod) [reply homeworks/12-mars-viewer-mobx/src/mars/heart-selected.svg]: Probably, I didn't check source code of SVGs. Removed them.

**Pull request merged**